docs/2.10.0/docs/canton/usermanual/monitoring.rst

```diff
@@ -175,7 +175,7 @@ Synchronization Domain Metrics
    This file is generated:
 .. include:: /canton/includes/generated/domain_metrics.rst.inc
 
-
+.. _health-metrics:
```

Comment:
You'll need a blank line between a label and the following section.

You'll have to apply this here and everywhere you defined the labels.